playground/src/App/index.tsx

```diff
@@ -22,7 +23,7 @@ import {
   ProgressButton,
   Hero,
   Card,
-  HorizontalRule,
+  // HorizontalRule,

```

Comment:
Commented code

Reply:
that's just the playground, yarn link was doin me dirty so I just commented it out but that doesnt affect the package